### DISCUSSION TITLE
Is there Video Tutorial / Discord / Telegram and Whatsapp Group?

### DISCUSSION THREAD

**Discussion by diogodine** [afpineda/OpenSourceSimWheelESP32#13] [Q&A] [answered]
Congratulations on the excellent work, the project is very complete and organized.

Is there a step-by-step tutorial video on YouTube or similar? Or group on Discord with discussions about the project? Any place that shows the projects completed and in progress? Group on Telegram or Whatsapp that I can join?

I looked, but I couldn't find it, if you know or have something please share it here, I would be very grateful.

Thank you and congratulations again on an excellent project.

**Comment by afpineda** [comment]
Hi. @diogodine.
Thanks for your support.

There is no social media around this project. This is not that big.
Use this forum to ask any question or concern.

**Comment by afpineda** [comment, marked as answer]
A Discord server is now available.
[https://discord.com/invite/sKQ4zK47DS](https://discord.com/invite/sKQ4zK47DS)
